src/server/visual-diff-reporter.js

```diff
@@ -1,8 +1,8 @@
-import { cpSync, mkdirSync, rmSync, writeFileSync } from 'fs';
-import { dirname, join } from 'path';
+import { cpSync, mkdirSync, rmSync, writeFileSync } from 'node:fs';
+import { dirname, join } from 'node:path';
```

Comment:
I'm always unsure when to use `fs` vs. `node:fs`... seems like you know more about this!

Reply:
I think they're supposed to do the same thing, but the `node:` prefix is newer and makes clear where it's coming from, instead of looking like a package. However, I added these because I was getting an error at some point, but didn't dig deeper since this fixed it.